wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,273 @@
 
 ### Skinning
 
-Set the preferred skin (from your own skin folder) that will be used instead if the skin is present in the another player's skin folder. If the skin is not present, a notice will pop-up and the player's default skin will be used. However, most people will just include the skin as part of the beatmap file so this setting is rarely used unless to decrease the filesize. Older beatmaps may uses this, hardly ever used nowadays.
+![The preferred skin is set as grungosu and there is no skin elements in the beatmapset folder as shown in the top-half. If the player have the skin name in the Skins folder, that skin will be used as shown in the bottom-half.][SS_ps]
+
+The `Preferred Skin` dropdown is used to set the preferred skin (from the player's `Skins` folder) to the defined skin name that will be used for the beatmap instead if the skin folder name is present in the player's `Skins` folder. If the skin folder name is not present, a notice will pop-up regarding the missing skin name and the player's default skin will be used. By default, `User's preference (No Change)` is used.
 
-An example of the usage of it would be [Beautiful Day - Bang! Bang! Bang! (-SiN-)](https://osu.ppy.sh/b/41820).
+Most mappers will just include the skin elements as part of the beatmap file so this setting is rarely used unless to decrease the filesize or is part of a beatmap project (where the skin may be provided on the project page itself). _Ancient_ beatmaps may use this.
 
-![Picture example here](img/SS_Skin_eg.jpg)
+An example of the usage would be [Beautiful Day - Bang! Bang! Bang! (-SiN-)][beatmap_PS] using the `grungosu` skin that _used_ to be available in the now-defunct `osume.exe` client (ancient _osu!_ updater with a preloaded list of skins available to be downloaded through the client).
 
-Advanced
---------
+## Advanced
 
-![Advanced section](img/SS_Advanced.jpg "Advanced section")
+![Advanced section][SS_adv]
 
 ### Stack Leniency
 
-osu! will automatically stacks notes that occur in the same place and close by in time. This is so that players can tell the objects apart.
+![Stack Leniency][SS_sl]
 
-The farther to the right that this slider is, the further apart in time stacking will occur. Rules dictate that if Stack Leniency is set so that stacking no longer occurs, you must manually offset the objects. This option is best left as it is *if* you *really* do not know what will happen and what you are doing.
+_osu!_ will automatically stacks notes that occur in the same place and close by in-time. This is so that players can tell the hit objects apart.
 
-Do note that stacked hit-circles will *move* towards the point where the stacking occurs.
+The farther to the right that this slider is, the further apart in-time stacking will occur. Rules dictate that if `Stack Leniency` is set so that stacking no longer occurs, the mapper must manually offset the hit objects. This option is best left as-is. Default value is `7`.
```

Comment:
The further to the right the slider is, the further apart in-time stacking will occur